CalculatorTests/CalculatorBrainTests.swift

```diff
@@ -0,0 +1,289 @@
+//
+//  CalculatorTests.swift
+//  CalculatorTests
+//
+//  Created by Wayne Rumble on 05/06/2017.
+//  Copyright © 2017 Wayne Rumble. All rights reserved.
+//
+
+import XCTest
+@testable import Calculator
+
+class CalculatorBrainTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testConstants() {
+        let symbolsAndValues = ["π": Double.pi, "e": M_E]
```

Comment:
Yea, if the test fails you have to investigate as the decription gives nothing.  I was concerned about repetition but i guess if it removes clarity then its better to have it?